docs/Getting Started/yo-generator.md

```diff
@@ -414,7 +414,7 @@ When the option is set to `true`, Keystone will scan the `updates` directory for
 
 - next - the method to call when the update has finished running (including any internal callbacks)
 
-Updates are ordered using [Semantic Versioning](semver.org), and Keystone will only run them once (successfully executed updates are stored in your database, in a collection called `app_updates`).
+Updates are ordered using [Semantic Versioning](https://semver.org), and Keystone will only run them once (successfully executed updates are stored in your database, in a collection called `app_updates`).
```

Comment:
Could split into two sentences for clarity. I also think Keystone will only apply once, if successful.